types/validator/index.d.ts

```diff
@@ -1051,20 +1051,25 @@ declare namespace validator {
      */
 
     interface strongPasswordOptions {
-        minLength?: number | undefined;
-        minLowercase?: number | undefined;
-        minUppercase?: number | undefined;
-        minNumbers?: number | undefined;
-        minSymbols?: number | undefined;
-        returnScore?: boolean | undefined;
-        pointsPerUnique?: number | undefined;
-        pointsPerRepeat?: number | undefined;
-        pointsForContainingLower?: number | undefined;
-        pointsForContainingUpper?: number | undefined;
-        pointsForContainingNumber?: number | undefined;
-        pointsForContainingSymbol?: number | undefined;
+        minLength?: number;
+        minLowercase?: number;
+        minUppercase?: number;
+        minNumbers?: number;
+        minSymbols?: number;
+        returnScore?: boolean;
+        pointsPerUnique?: number;
+        pointsPerRepeat?: number;
+        pointsForContainingLower?: number;
+        pointsForContainingUpper?: number;
+        pointsForContainingNumber?: number;
+        pointsForContainingSymbol?: number;
```

Comment:
```suggestion
        minLength?: number | undefined;
        minLowercase?: number | undefined;
        minUppercase?: number | undefined;
        minNumbers?: number | undefined;
        minSymbols?: number | undefined;
        returnScore?: boolean | undefined;
        pointsPerUnique?: number | undefined;
        pointsPerRepeat?: number | undefined;
        pointsForContainingLower?: number | undefined;
        pointsForContainingUpper?: number | undefined;
        pointsForContainingNumber?: number | undefined;
        pointsForContainingSymbol?: number | undefined;
```
those were there for TS 4.4. exact optional property types I think:
https://devblogs.microsoft.com/typescript/announcing-typescript-4-4/#exact-optional-property-types
and the native package supports strict 'undefined' for options:
https://github.com/validatorjs/validator.js/blob/63b61629187a732c3b3c8d89fe4cacad890cad99/src/lib/util/merge.js#L3

Reply:
Yikes! Just learnt something new! Thanks